### PR TITLE
Remove unnecessary properties from Blazor WebAssembly project file.

### PR DESCRIPTION
**Pull request by danroth27**

### CODE REVIEW

File: src/Components/Blazor/Templates/src/content/BlazorWasm-CSharp/Client/BlazorWasm-CSharp.Client.csproj [hunk 1]
```diff
@@ -2,8 +2,6 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <OutputType>Exe</OutputType>
-    <LangVersion>7.3</LangVersion>
```
Comment (pranavkm) [inline src/Components/Blazor/Templates/src/content/BlazorWasm-CSharp/Client/BlazorWasm-CSharp.Client.csproj]: Ahh cool. The compiler defaults to C# 7.3 for ns2.0. 👍

**Review by pranavkm — APPROVED** [review]

File: src/Components/Blazor/Templates/src/content/BlazorWasm-CSharp/Client/BlazorWasm-CSharp.Client.csproj [hunk 2]
```diff
@@ -2,8 +2,6 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <OutputType>Exe</OutputType>
```
Comment (SteveSandersonMS) [inline src/Components/Blazor/Templates/src/content/BlazorWasm-CSharp/Client/BlazorWasm-CSharp.Client.csproj]: Isn't OutputType=Exe what we want for an application? There does have to be a static Program.Main entrypoint in order for it to run. It's good if the compiler stops you from building without that.

Comment (danroth27) [reply src/Components/Blazor/Templates/src/content/BlazorWasm-CSharp/Client/BlazorWasm-CSharp.Client.csproj]: I just tried it, and you still get a compilation error in the client project if you don't have an entry point even without `<OutputType>Exe</OutputType>`.

Comment (SteveSandersonMS) [reply src/Components/Blazor/Templates/src/content/BlazorWasm-CSharp/Client/BlazorWasm-CSharp.Client.csproj]: Oh, so `Exe` is the default? If so, fine with me!

Comment (rynowak) [reply src/Components/Blazor/Templates/src/content/BlazorWasm-CSharp/Client/BlazorWasm-CSharp.Client.csproj]: Exe is the default for `.Web`

**Review by SteveSandersonMS — APPROVED** [review]

**Pull request merged**